src/containers/dom-element-renderer.jsx

```diff
@@ -31,14 +32,21 @@ class DOMElementRenderer extends React.Component {
         // Look at me, I'm the React now!
         Object.assign(
             this.props.domElement,
-            omit(this.props, ['domElement', 'children'])
+            omit(this.props, ['domElement', 'children', 'style'])
         );
+
+        // Convert react style prop to dom element styling.
+        if (this.props.style) {
+            this.props.domElement.style = Style.string(this.props.style);
+        }
+
         return <div ref={this.setContainer} />;
     }
 }
 
 DOMElementRenderer.propTypes = {
-    domElement: PropTypes.instanceOf(Element).isRequired
+    domElement: PropTypes.instanceOf(Element).isRequired,
+    style: PropTypes.object // eslint-disable-line react/forbid-prop-types
```

Comment:
Believe it or not we have something for this! https://github.com/LLK/scratch-gui/blob/5b082e29ecf81386e122ecc1c0386ba2daecba7a/src/components/box/box.jsx#L124

Reply:
Oh cool! c3b3bb2